src/CSET/operators/plot.py

```diff
@@ -263,7 +263,7 @@ def spatial_contour_plot(
         plot_filename = f"{filename.rsplit('.', 1)[0]}_{sequence_value}.png"
         time_coord = cube_slice.coord(sequence_coordinate)
         times = time_coord.units.num2date(time_coord.points[0])
-        title = f"{("{:04d}{:02d}{:02d} {:02d}:{:02d}".format(times.year, times.month, times.day, times.hour, times.minute))}"
+        title = f"{("{:4d}{:02d}{:02d} {:02d}:{:02d}".format(times.year, times.month, times.day, times.hour, times.minute))}"
```

Comment:
```suggestion
        title = times.isoformat()
```
The issue causing the tests to fail here is that you have unescaped `"` inside the `"` defining the string boundaries. This is allowed from python 3.12, but not before.

However as we have a datetime object to start with, we can just use the provided method to get a string format of it.